src/main/java/net/snowflake/ingest/streaming/internal/FlushService.java

```diff
@@ -498,7 +510,7 @@ BlobMetadata buildAndUpload(String filePath, List<List<ChannelData>> blobData)
                 // The start offset will be updated later in BlobBuilder#build to include the blob
                 // header
                 .setChunkStartOffset(curDataSize)
-                .setChunkLength(encryptedCompressedChunkDataSize)
+                .setChunkLength(chunkLength)
```

Comment:
could you add a comment explain why we're using the compressedChunkLength, but not encryptedCompressedChunkDataSize